src/Symfony/Component/HttpKernel/Fragment/RoutableFragmentRenderer.php

```diff
@@ -31,7 +31,7 @@ abstract class RoutableFragmentRenderer implements FragmentRendererInterface
      *
      * @see FragmentListener
      */
-    public function setFragmentPath($path)
+    public function setFragmentPath(string $path)
```

Comment:
`@param` is useless now (please check other place :) )

Reply:
Removed and the other ones too :)